### ISSUE TITLE
Call to a member function getResizedImageUrl() on null

### ISSUE THREAD

**Issue by sadortun** [orocrm/platform#665]
Hi Oro Team

/cc @soloma88 

When viewing a Lead Actrivity Comment, 
```
Notice</b>:  Undefined property: Oro\Bundle\CommentBundle\Entity\Manager\CommentApiManager::$attachmentManager in /var/www/orocrm/vendor/oro/platform/src/Oro/Bundle/CommentBundle/Entity/Manager/CommentApiManager.php</b> on line <b>238</b><br />
Fatal error</b>:  Uncaught Error: Call to a member function getResizedImageUrl() on null in /var/www/orocrm/vendor/oro/platform/src/Oro/Bundle/CommentBundle/Entity/Manager/CommentApiManager.php:239
```

In  https://github.com/orocrm/platform/commit/96d100f4663c1e9a1e5848f0d1a05144f27fa793#diff-59732a79388412be1cf9270b3ccde35dL46

`$attachmentManager` was renamed to  `$attachmentProvider` but the reference https://github.com/orocrm/platform/blob/4c2271ce5c7bb37745bcc5dfd6083731a9741b67/src/Oro/Bundle/CommentBundle/Entity/Manager/CommentApiManager.php#L238
was left behind

**Comment by mkudelya** [comment]
Hi @sadortun 

Thank you for reporting the issue! The same issue has already been discussed on the [forum](https://www.orocrm.com/forums/topic/php-fatal-error-call-to-a-member-function-getresizedimageurl-on-null).

**Comment by mkudelya** [comment]
The issue has been fixed in version 2.1.0. Thank you for pointing out the issue!
